practical/jython/load_thumbnails.jy

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env jython
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+#  Copyright (C) 2018 University of Dundee. All rights reserved.
+#
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# ------------------------------------------------------------------------------
+
+# This script loads the thumbnails associated to images and open them in
+# ImageJ. The thumbnails are turned into greyscale images in order to be
+# viewed in ImageJ
+# The purpose of the script is to be used in the Scripting Dialog
+# of Fiji (File > New > Script). 
+# Select Python as language in the Scripting Dialog.
+# Error handling is omitted to ease the reading of the script but
+# this should be added
```

Comment:
here this line is commented out, which signals to me that the two other occassions mentioned above are bugs